Makefile

```diff
@@ -186,8 +186,9 @@ $(foreach DB,$(DATABASES),$(eval $(call database,$(DB))))
 ARTIX_PARTS=artix7_50t artix7_200t
 ZYNQ_PARTS=zynq7010
 KINTEX_PARTS=
+SPARTAN_PARTS=spartan7_50
```

Comment:
These are actually needed to build extra parts. By extra part is meant everything else apart from the bits documentation, e.g. tile grid, pips and wires, etc.

For instance, the main part for the artix7 that is used for the bit documentation (e.g. the one that runs the fuzzers) is the 100T: https://github.com/SymbiFlow/prjxray/blob/master/settings/artix7.sh

The `50t` and `200t` are added to generate the tilegrid and tileconn files as well.

Similarly, for the spartan7 we'd need to have a main part, in this case the `50`, which shouldn't be added here as an extra part.